app/components/environment_banner_component.html.erb

```diff
@@ -0,0 +1,5 @@
+<header class="environment-banner">
+  <div class="govuk-width-container">
+    This is staging. Only process test cases on this version of BoPS
```

Comment:
Lower case 'o' 🙌 😄

Reply:
I hope they appreciate the quality that has gone into this 🤔